src/KeyFrame.cc

```diff
@@ -391,7 +426,6 @@ void KeyFrame::MakeKeyFrame_Rest()
     {
       std::vector<CVD::ImageRef> vMaxCornersTemp;
       CVD::fast_nonmax(lev.image, lev.vCorners, lev.nFastThresh, vMaxCornersTemp);
```

Comment:
Why are these being done again? Or is it an if on buffering? We're only looking for fast features in an image once, right?